parcels/fieldfilebuffer.py

```diff
@@ -72,7 +72,7 @@ def parse_name(self, name):
                     name = nm
                     break
         if isinstance(name, list):
-            raise IOError('None of variables in list found in file')
+            raise OSError('None of variables in list found in file')
```

Comment:
Why does IOError not exist anymore? This error really is about the input files, not about the OS?

Reply:
As of Python 3.3, [PEP3151](https://docs.python.org/3/whatsnew/3.3.html#pep-3151) changed such that `IOError` and `EnvironmentError` are only aliases for `OSError`. They are only supported for backwards compatability, keeping it in the code doesn't provide more informative error messages.

```
Python 3.10.6 | packaged by conda-forge | (main, Aug 22 2022, 20:36:39) [GCC 10.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> raise EnvironmentError("Invalid environment!")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OSError: Invalid environment!
>>>
```